src/seedu/addressbook/AddressBook.java

```diff
@@ -586,6 +607,22 @@ private static void executeExitProgramRequest() {
         exitProgram();
     }
 
+
+    /**
+    * Displays all persons in the address book to the user; in alphabetical order.
+    */
```

Comment:
Glad to see that header comments are added =).
Maybe it can be improved by removing the `;`.